specs/capri/FileShare_Design.md

```diff
@@ -0,0 +1,737 @@
+# Title
+
+**Author(s)**:  [Shruthi MN](https://github.com/Shruthi-1MN), [Pravin Ranjan](https://github.com/PravinRanjan10)
+
+## Summary
+
+A share is a remote, mountable file system. You can mount a share to and access a share from several hosts by several users at a time. we can create a share, list shares, and show information for, update, and delete a share.To create a share, specify one of these supported protocols:
+* NFS
+* CIFS
+
+## Motivation
+
+Multiple users can access the same file share mount point.
+Data that would otherwise be duplicated on each client can be kept in a single location and accessed by client on the network.
+
+### Goals
+
+* Facilitating File Share Service by providing Standard API to manage multiple vendors, simplify File Share API definition  
+* File share across the users based on access capability
+* Scope for now is to support only NFS, SMB file share protocols
+
```

Comment:
Add a goal for profiles.
Add a goal to write a Manila driver and a default NFS/SMB driver.
Add a goal to provide CLIs.

Reply:
All points added to goal.

Reply:
Added goal for profile